docs/database_schema_diagrams/source/entity_network_details.json

```diff
@@ -1,6 +1,6 @@
 {
-  "tooltip": "Tables specifically related to core entities",
-  "_comment": "It shows tables unique to some entities; That is beyond alias, collection, disambiguation comment, edit, link, subscription, rating, tag.",
+  "tooltip": "Tables uniquely related to primary entities",
```

Comment:
should that be "primary entity types"?

Reply:
yeah, probably, changed in commit https://github.com/metabrainz/musicbrainz-server/pull/2987/commits/ff41bd6b09f2a563e01c78e2b15e07a598d230d7